checkov/common/checks_infra/solvers/attribute_solvers/is_empty_attribute_solver.py

```diff
@@ -0,0 +1,24 @@
+from typing import List, Optional, Any, Dict
+
+from checkov.common.graph.checks_infra.enums import Operators
+from checkov.common.checks_infra.solvers.attribute_solvers.base_attribute_solver import BaseAttributeSolver
+
+
+class IsEmptyAttributeSolver(BaseAttributeSolver):
+    operator = Operators.IS_EMPTY
+
+    def __init__(self, resource_types: List[str], attribute: Optional[str], value: Any) -> None:
+        super().__init__(resource_types=resource_types, attribute=attribute, value=value)
+
+    def _get_operation(self, vertex: Dict[str, Any], attribute: Optional[str]) -> bool:
+        attr = vertex.get(attribute)  # type:ignore[arg-type]  # due to attribute can be None
+
+        # if this value contains an underendered variable, then we cannot evaluate the check,
+        # so return True (since we cannot return UNKNOWN)
+        if self._is_variable_dependant(attr, vertex["source_"]):
+            return True
+
+        if isinstance(attr, (list, dict)):
+            return len(attr) == 0
+
+        return attr == ""
```

Comment:
need to import this
```python
from collections.abc import Sized
```
then you can use
```suggestion
        if isinstance(attr, Sized):
            return len(attr) == 0

        return False
```
I think default `False` for anything, which can't be checked for being empty sounds reasonable 🙂

Reply:
@gruebel how would it work for empty string though?  :) 
if `attr = ''` then we will pass this check, no?

Reply:
Great point!!
Now learned that `Sized` also works for strings!